Sources/Components/BottomSheet/Animator/SpringAnimator.swift

```diff
@@ -0,0 +1,100 @@
+//
+//  Copyright © 2018 FINN.no. All rights reserved.
+//
+
+import UIKit
+
+extension SpringAnimator {
+    enum State {
+        case animating, paused, cancelled, stopped
+    }
+}
+
+class SpringAnimator: NSObject {
+
+    // Spring properties
+    let damping: CGFloat
+    let stiffness: CGFloat
+
+    // View properties
+    private var velocity = 0.0 as CGFloat
```

Comment:
Nitpicking here, but I prefer `private var velocity: CGFloat = 0`

Reply:
Hehe, I can change it